From bf78c8ef0611d40e78360647c83ae0e0ecf2f92e Mon Sep 17 00:00:00 2001
From: mac <zjp15083791743@163.com>
Date: Tue, 29 Oct 2024 04:11:30 +0800
Subject: [PATCH] =?UTF-8?q?=E6=A0=87=E7=AD=BE=E5=8A=9F=E8=83=BD?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 git操作测试 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/git操作测试 b/git操作测试
index e40d919..73211cc 100644
--- a/git操作测试
+++ b/git操作测试
@@ -28,4 +28,4 @@ mac_dev分支是提交时直接勾选立即推送变更到 origin/mac_dev好，
 便于团队协作，其他人可以看到每次提交的信息，理解代码的变化。
 
 sourcetree标签:
-
+输入想要的标签名，标签代表这个代码状态或者功能
-- 
2.46.0

